### PR TITLE
Add AddExtra support.

### PR DESCRIPTION
**Pull request by Colstuwjx**
As we need some extra context to debug problems, AddExtra is needed.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -510,7 +523,7 @@ func (client *Client) worker() {
 // Capture asynchronously delivers a packet to the Sentry server. It is a no-op
 // when client is nil. A channel is provided if it is important to check for a
 // send's success.
-func (client *Client) Capture(packet *Packet, captureTags map[string]string) (eventID string, ch chan error) {
+func (client *Client) Capture(packet *Packet, captureTags map[string]string, extra map[string]interface{}) (eventID string, ch chan error) {
```
Comment (mattrobenolt) [inline client.go]: This is changing the API. :(

**Comment by Colstuwjx** [comment]
Hi @mattrobenolt 

It seems that we have some PR( like #74 ) that try to add extra method or full extra context support.
We have been waited it for one year ago, and I think it's time to merge extra context support to upstream.

**Comment by jheth** [comment]
Can this be closed now that #190 is merged?

**Comment by Colstuwjx** [comment]
Yes, I think the differences between adding extra within `Capture...` method and `NewPacketWithExtra...` is  an acceptable change, so cool for that!

**Pull request closed without merging**